src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala

```diff
@@ -120,4 +119,11 @@ class FailureHandlerSpec extends LambdaSpec {
       case _ => fail()
     }
   }
+
+  it should "convert a transaction declined error from Zuora to an appropriate CheckoutFailureReason" in {
+    val failureHandler = new FailureHandler()
+    val reason = failureHandler.toCheckoutFailureReason(ZuoraError("TRANSACTION_FAILED", "Transaction declined.do_not_honor - Your card was declined."))
+    reason should be(PaymentMethodUnacceptable)
+  }
+
```

Comment:
It would be good to check the reason returned in all the other test cases in this spec.